<launch>

  <node name="maVisu" pkg="image_view" type="image_view" output="screen" >
    <remap from="image" to="/ardrone/front/image_raw" />
  </node>

  <node name="fd" pkg="tp_ros_ensta_u2is" type="fd.py">
    <remap from="image_topic_in" to="/ardrone/front/image_raw" />
  </node>

  <node name="maVisuOut" pkg="image_view" type="image_view" output="screen" >
    <remap from="image" to="/image_topic_out" />
  </node>

</launch>
